mysite/_config/injector.yml

```diff
@@ -27,22 +27,25 @@ Injector:
   BuildAddonsTask:
     constructor:
       - %$AddonBuilder
-
 ---
 Only:
   environment: 'live'
+After:
+  - '#elastica'
 ---
 Injector:
   SilverStripe\Elastica\ElasticaService:
     constructor:
       - %$Elastica\Client
-      - 'addons'
+      - addons
 ---
 Except:
   environment: 'live'
+After:
+  - '#elastica'
 ---
 Injector:
   SilverStripe\Elastica\ElasticaService:
     constructor:
       - %$Elastica\Client
-      - 'addons-dev'
\ No newline at end of file
+      - sample-index
```

Comment:
I think we have an issue here, the module should be able to set the index name rather than accepting the default. This will cause issues when we go to production. This shouldn't have been merged really until we addressed this.